### PR TITLE
Added table exports to Externs

### LINKED ISSUE #17: No support for tables in Instance.Externs
If Module.Exports.Tables contains any values, the exception NotSupportedException with the message "Unsupported extern type." is thrown in the Externs constructor when Host.Instantiate is called

### PR DESCRIPTION
**Pull request by jlgaffney**
Fixes #17

Include table exports into the Externs instance

### CODE REVIEW

File: tests/TableExportsTests.cs [hunk 1]
```diff
@@ -37,6 +37,33 @@ public void ItHasTheExpectedNumberOfExportedTables()
             GetTableExports().Count().Should().Be(Fixture.Module.Exports.Tables.Count);
         }
 
+        [Fact]
+        public void ItCreatesExternsForTheGlobals()
```
Comment (peterhuene) [inline tests/TableExportsTests.cs:41]: Should this be `ItCreatesExternsForTheTables`?

File: src/Externs/ExternTable.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    public class ExternTable
+    {
+        internal ExternTable(TableExport export, IntPtr table)
+        {
+            _export = export;
+            _table = table;
+        }
+
+        /// <summary>
+        /// The name of the WebAssembly memory.
```
Comment (peterhuene) [inline src/Externs/ExternTable.cs:18]:
```suggestion
        /// The name of the WebAssembly table.
```

**Comment by jlgaffney** [comment]
Ah yes, sorry about that

File: tests/TableExportsTests.cs (hunk 1 above)
Comment (jlgaffney) [reply tests/TableExportsTests.cs]: Fixed it, should be good now

**Comment by peterhuene** [comment]
@jlgaffney thanks for this!

**Pull request merged**